code/modules/mob/living/brain/brain_item.dm

```diff
@@ -481,6 +481,10 @@
 /obj/item/organ/brain/felinid //A bit smaller than average
 	brain_size = 0.8
 
+/obj/item/organ/brain/felinid/on_mob_insert(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+	organ_owner.gain_trauma(/datum/brain_trauma/mild/phobia/cucumbers)
```

Comment:
`mob.gain_trauma` just calls `brain.brain_gain_trauma`
probably just calling `brain_gain_trauma` in `Initialize` would have a similar effect